### DISCUSSION TITLE
My Intermediate Java class requires GitHub Student Developer Pack, but there is no upload option.

### DISCUSSION THREAD

**Discussion by Sweek9** [community/community#31884] [GitHub Education]
Edit: 
Someone sent me the link to send a "ticket" to GitHub, so I'm trying that too. 

Start Of Original:
I've tried multiple browsers on multiple devices, and there is never an upload option. Reading other discussion posts I know this is an intended security feature, but that doesn't change the fact that it's making me get an F in my class. My last resort was this Aspire laptop I checked out from the San Diego Miramar College Library, and it still doesn't work. 

If this security feature is being triggered on a school computer, I can confidently say it's flawed. Furthermore the HTML camera app I'm forced to use has no focus and low resolution regardless of camera quality, and all photos are rejected before I can even submit them. I can't find anything in the FAQ, and this was the only other form of support I could find on GitHub. Any ideas? 

This is what I'm seeing:
![image](https://user-images.githubusercontent.com/2678959/188288160-4bdbdef7-2957-46c0-8bc3-b1b4de77b418.png)
![image](https://user-images.githubusercontent.com/2678959/188288370-956f61ee-ce31-48e2-a6c1-7ea70462e9f4.png)

This is what I'm trying to submit:
![image](https://user-images.githubusercontent.com/2678959/188288242-0cee8361-8d2a-42f0-b8b6-4b50a736c003.png)

**Comment by riazmahmud1** [comment]
I would suggest you try one of the following two options below:

    Submit your application using one of the other documents seen below (choose one besides the student ID):
![--](https://user-images.githubusercontent.com/112637573/188302717-c95faffc-b72e-4036-b0a5-ff06a2e23df0.png)

**Reply by Sweek9** [reply]
Thank you for the response engineerriazmahmud. Unfortunately there is no upload option, and regardless of what I take a photo of the image is rejected for being too blurry, not even allowing me to submit it. Their built in camera app doesn't focus the camera, and caps resolution at a low setting.

**Comment by melvincwng** [comment]
@Sweek9 Hey Moorgan, can you try one of the 3 suggestions which I've previously provided a user over here: https://github.com/orgs/community/discussions/31852

Hope this answers and resolves your issue 😊

**Reply by Sweek9** [reply]
Thank you for the response melvincwng. Unfortunately there is no upload option, and regardless of what I take a photo of the image is rejected for being too blurry, not even allowing me to submit it. Their built in camera app doesn't focus the camera, and caps resolution at a low setting. I also already sent a ticket before I got your response, and it was immediately closed without any response.
